tests/src/test/scala/com/nvidia/spark/rapids/AdaptiveQueryExecSuite.scala

```diff
@@ -294,6 +297,81 @@ class AdaptiveQueryExecSuite
     }, conf)
   }
 
+  test("Avoid transitions to row when writing to Parquet") {
```

Comment:
It would be good to add a negative test where an adaptive plan appears but it is _not_ a columnar plan and therefore we should not remove the row-to-columnar transition because it is necessary.